Makefile

```diff
@@ -153,6 +154,28 @@ test_extension_ci: $(SO_FILE) $(TEST_FILES) $(TEST_STUB_FILES)
 	$(RUN_TESTS_CMD) -d extension=$(SO_FILE) -m -s $$TEST_PHP_OUTPUT $(BUILD_DIR)/$(TESTS) && ! grep -e 'LEAKED TEST SUMMARY' $$TEST_PHP_OUTPUT; \
 	)
 
+build_appsec_ext: $(SO_FILE) clean_appsec_ext
```

Comment:
We can skip the _ext suffix, as this is the common operation I guess

Reply:
I'm not sure I'm keeping this in the Makefile but the objective was to build the extension and helper separately, although it's not clear to me now in which circumstance we'd use this Makefile when CMake would be faster, as it is properly incremental.